charts/kafkametriccollectoragent/values.yaml

```diff
@@ -0,0 +1,27 @@
+replicaCount: 1
+
+image:
+  repository: insightfinderinc/kafkametriccollectoragent
+  pullPolicy: IfNotPresent
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+podLabels: {}
+
+resources:
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# - The configuration of application.yml
+config: {}
```

Comment:
Can you put the example of `application.yml `here?
So that users don't need to look for the example elsewhere